swift/ql/lib/codeql/swift/security/CleartextLoggingExtensions.qll

```diff
@@ -99,16 +99,16 @@ private class LoggingSinks extends SinkModelCsv {
       [
         ";;false;print(_:separator:terminator:);;;Argument[0..2];log-injection",
         ";;false;print(_:separator:terminator:toStream:);;;Argument[0..2];log-injection",
-        ";;false;print(_:separator:terminator:to:);;;Argument[0..2];log-injection",
         ";;false;debugPrint(_:separator:terminator:);;;Argument[0..2];log-injection",
-        ";;false;debugPrint(_:separator:terminator:to:);;;Argument[0..2];log-injection",
         ";;false;dump(_:name:indent:maxDepth:maxItems:);;;Argument[0..1];log-injection",
-        ";;false;dump(_:to:name:indent:maxDepth:maxItems:);;;Argument[0];log-injection",
-        ";;false;dump(_:to:name:indent:maxDepth:maxItems:);;;Argument[2];log-injection",
+        ";;false;assert(_:_:file:line:);;;Argument[1];log-injection",
+        ";;false;assertionFailure(_:file:line:);;;Argument[0];log-injection",
+        ";;false;precondition(_:_:file:line:);;;Argument[1];log-injection",
+        ";;false;preconditionFailure(_:file:line:);;;Argument[0];log-injection",
         ";;false;fatalError(_:file:line:);;;Argument[0];log-injection",
         ";;false;NSLog(_:_:);;;Argument[0..1];log-injection",
         ";;false;NSLogv(_:_:);;;Argument[0..1];log-injection",
-        ";;false;vfprintf(_:_:_:);;;Agument[1..2];log-injection",
+        ";;false;vfprintf(_:_:_:);;;Argument[1..2];log-injection",
```

Comment:
We should probably have a ql-for-ql query that spots these kinds of things 🤔

Reply:
Yeah.  I thought I saw some code somewhere that verifies the formatting is correct, but (a) I'm not sure it would catch a typo like this and (b) I don't think it's run as a PR check.  On the positive side, the tests found it.